README.md

```diff
@@ -54,7 +54,18 @@ git clone git@github.com:vantage-ola/track-now.git
 git checkout -b <your branch name>
 ```
 
-3. To access the mongo db database to run the flask api , create a local environment `/backend/.env` and copy the variables.(READ_ACCESS_ONLY)
+3. Install dependencies:
+```shell
+pip3 install DEPENDENCIES.txt
+```
+
+4. Create and activate virtualenvironment.
```

Comment:
creating a virtual env should come first before installing dependencies... this should be no 3, then installing dependencies should be no 4...you can also add installation for windows users also ...

like
  - On Windows:

     ```bash
     .\venv\Scripts\activate
     ```

   - On macOS and Linux:

     ```bash
     source venv/bin/activate
     ```